include/Types.h

```diff
@@ -18,9 +18,19 @@
 #pragma once
 
 #include <cstdint>
+#include <string.h>
+//#include <exception.h>
+#include <bits/stdc++.h>
```

Comment:
It's usually not a good idea to include all the stl libraries in one go. Prefer to include as little as possible.